lib/db/schema/patch-020-021.sql

```diff
@@ -0,0 +1,35 @@
+CREATE PROCEDURE `sessionDevice_1` (
+  IN `inTokenId` BINARY(32)
+)
+BEGIN
+  SELECT
+    t.tokenData,
+    t.uid,
+    t.createdAt,
+    t.uaBrowser,
+    t.uaBrowserVersion,
+    t.uaOS,
+    t.uaOSVersion,
+    t.uaDeviceType,
+    t.lastAccessTime,
+    a.emailVerified,
+    a.email,
+    a.emailCode,
+    a.verifierSetAt,
+    a.locale,
+    a.createdAt AS accountCreatedAt,
+    d.id AS deviceId,
+    d.name AS deviceName,
+    d.type AS deviceType,
+    d.createdAt AS deviceCreatedAt,
+    d.callbackURL,
+    d.callbackPublicKey
+  FROM
+    sessionTokens t
+    LEFT JOIN accounts a ON t.uid = a.uid
+    LEFT JOIN devices d ON t.tokenId = d.sessionTokenId
+  WHERE
+    t.tokenId = inTokenId;
```

Comment:
although both queries produce the same optimized query according to `SHOW WARNINGS`:

```
... from `fxa`.`sessionTokens` `t` left join `fxa`.`accounts` `a` on(multiple equal('O�k}�CW�Oeˋ$��', 'O�k}�CW�Oeˋ$��')) left join `fxa`.`devices` `d` on((`fxa`.`d`.`sessionTokenId` = 0xe6b5b3eea6e05587)) where 1
```